controller/volume_attachment_controller.go

```diff
@@ -488,6 +492,7 @@ func (vac *VolumeAttachmentController) shouldDoDetach(va *longhorn.VolumeAttachm
 	}
 
 	if len(currentAttachmentTickets) == 0 {
+		log.Infof("Should detach volume %v because there is no attachment ticket", vol.Spec.NodeID)
```

Comment:
NIT: "there is no **matching** attachment ticket"

Reply:
Updated. Thank you.